packages/react-components/src/components/TypingIndicator.tsx

```diff
@@ -147,6 +147,13 @@ const getUsersElement = (
 ): JSX.Element => {
   const userElements: JSX.Element[] = [];
   typingUsers.forEach((user, index) => {
+    if (user.displayName) {
+      let truncatedDisplayName = user.displayName;
+      if (truncatedDisplayName.length > 50) {
+        truncatedDisplayName = truncatedDisplayName.substring(0, 50) + '...';
+      }
+      user.displayName = truncatedDisplayName;
```

Comment:
does this logic make sense that each user can have up to 50 characters? so if 2 people have long name total typing indicator name would be 100 char long? Just something to think about but this is a start

Reply:
Yep I discussed this with Alex as well. Originally the plan was to have it based on the width %. But it would be difficult and require a refactor of our typing indicator. So instead we will do a char limit on the display name to 50.